contracts/leaser/src/msg.rs

```diff
@@ -55,6 +55,10 @@ pub enum ExecuteMsg {
     /// It migrates the next batch of up to `max_leases` number of Lease instances
     /// and emits the status as specified in `MigrateLeases`.
     MigrateLeasesCont { key: Addr, max_leases: MaxLeases },
+    DetachClosed {
+        max_leases: MaxLeases,
+        next_key: Option<Addr>,
```

Comment:
let's keep the consistency with the above variants of the argument definition:
- flip the order
- `next_key` -> ``key`

Reply:
Applying the change, but I'd like to remind you that to the JSON the order won't matter and that the optional field will be first. Just doesn't seem natural.

Reply:
exactly, the format would not be an obstacle ... the goal is to reach better consistency among the variants on the API level